MapboxNavigation/NavigationOptions.swift

```diff
@@ -29,6 +29,13 @@ open class NavigationOptions: NSObject, NavigationCustomizable {
     */
     @objc open var voiceController: RouteVoiceController?
     
+    /**
+     The view controller to embed into the top section of the UI.
+     
+     If this property is set to `nil`, a `TopBannerViewController` is created and embedded in the UI. This property is set to `nil` by default.
+     */
+    @objc open var topBanner: ContainerViewController?
```

Comment:
Let’s minimize the use of words that lock us into a particular layout. For instance, after we improve the layout in landscape orientation (#1139), the “top” banner will be off to the side rather than on top. How about `primaryBanner`? Or would that be perilously close to the name of some properties and classes in MapboxDirections.swift?

Reply:
Posterity: Talked with minh about this,  we agreed that it would be better to wait to tackle this until we refactor landscape in earnest. (#1139).